config/sync/core.extension.yml

```diff
@@ -116,6 +116,7 @@ module:
   purge_queuer_coretags: 0
   purge_tokens: 0
   recurring_events: 0
+  redirect: 0
```

Comment:
The Redirect module contains a submodule especially for creating redirects for 404s. I think we should discuss enabling this with the business. I think it would be helpful to administrators in the migration process after going live.

![Monosnap Extend | DPL CMS | Logged in 2024-04-22 06-36-47](https://github.com/danskernesdigitalebibliotek/dpl-cms/assets/73966/b2c6dce4-eee0-4b75-b92a-27cfffd569f8)